index.js

```diff
@@ -62,8 +62,8 @@ const expressions = {
     handler: (match, query) => `${BASE_SCC_URL}/search?q=${getQueryFromParams(match[0], query)}${getIndexMapping(match[3])}`
   },
   patroninfoReg: {
-    expr: /\/patroninfo[^\/]\/(\d+)/,
-    handler: match => `${BASE_SCC_URL}?fakepatron=${match[1]}`
+    expr: /\/patroninfo\/(\d+)/,
```

Comment:
This wouldn't match the `~S1` in, e.g., `https://legacycatalog.nypl.org/patroninfo~S1/4746737/items`, right?
Also probably don't need to capture the `\d+` if we're not using it later...

Reply:
(if that's a case you want to cover you might want something like `/\/patroninfo(?:[^/]*)\/\d+/`)
((I'd also always include a beginning of string anchor `^` where possible))

Reply:
Good point. Since it turns out we don't need the patron id, it looks like we could just use `\/patroninfo` as the expression. @thisisstephenbetts does that sound OK?

Reply:
Good point. I guess so, but I suggest you look through the logs to make sure there aren't other use cases of `patroninfo`. But I believe that you are correct.

Reply:
(I mean, I'd prefer `^\/patroninfo` if that works, but... :-) )

Reply:
I think that works. Fixed.

Reply:
@danamansana — you still going to remove the `\d+`? Let me know and I'll approve.

Reply:
Sorry, I thought I'd pushed that. Should be removed now.